packages/react-components/theme-designer/src/components/TokenBoxes/TokenBoxes.tsx

```diff
@@ -1,14 +1,12 @@
 import * as React from 'react';
 import { makeStyles, shorthands } from '@griffel/react';
-import { Button, useId, Input, Switch, Caption1, tokens } from '@fluentui/react-components';
+import { Button, useId, Input, Caption1, tokens } from '@fluentui/react-components';
```

Comment:
_nit:_ Order alphabetically

```suggestion
import { tokens, useId, Button, Caption1, Input } from '@fluentui/react-components';
```